helpers/repositories.json

```diff
@@ -1 +1 @@
+["yurinnick/folding-at-home:latest", "kadena/chainweb-node:latest", "zelcash/dibi-fetch:latest", "zelcash/rates-api:latest", "zelcash/kadena-chainweb-node:2.7", "presearch/node:latest", "bharathshetty4/supermario:latest", "zedeus/nitter:latest", "bsord/tetris:latest", "runonflux/rosetta-server:latest", "itzg/minecraft-server:latest", "runonflux/polkadot-docker:latest", "littlestache/cors-anywhere:latest", "littlestache/minecraft:latest", "littlestache/fluxwhitepaper:latest", "treyyoder/quakejs:latest", "littlestache/bitcoinwhitepaper:latest", "littlestache/pokerth:latest", "firoorg/firod:latest", "littlestache/kdachainwebpaper:latest", "littlestache/kdapublicsumpaper:latest", "littlestache/quakejs:latest", "littlestache/spacecompany:latest", "littlestache/hexgl:latest", "littlestache/os13k:latest", "littlestache/evolve:latest", "littlestache/shapezio:latest", "littlestache/lightbike:latest", "littlestache/civclicker:latest", "littlestache/level13:latest", "littlestache/dwarfs:latest", "littlestache/bounceback:latest", "littlestache/syncpaint:latest", "littlestache/thehouse:latest", "littlestache/prestigetree:latest", "littlestache/progressknight:latest", "runonflux/website:latest", "littlestache/radiusraid:latest", "ch4itu/quotations:latest", "basco64/fluxsnakegame:latest", "basco64/testfps:latest", "martintale/go-away:latest", "vertfromage/jeffs-nightmare:latest", "lnoird/dropalo-web:latest", "ethereum/client-go:v1.10.11", "littlestache/btcwhitepaperaudio:latest", "littlestache/fluxwhitepapertranslations:latest", "zzzpotato/zercwebsite:latest", "littlestache/helloworld:latest", "wordpress:latest", "mysql:5.7", "ethereum/client-go:stable", "shatayen/defiportfoliomanager_stream", "shatayen/defiportfoliomanager_backend", "shatayen/defiportfoliomanager_frontend"]
```

Comment:
We need to specify a tag of the repository namespace. Eg. :latest. Please correct.

Reply:
Sure, forgot... Next commit has it.